integration_tests/snaps/pip-requirements-file/snapcraft.yaml

```diff
@@ -11,7 +11,9 @@ confinement: strict
 parts:
   python2:
     plugin: python2
+    source: .
```

Comment:
Well no, I guess not since we have `python-packages`, but could we toss an obvious error if `requirements` is used without `source`?